packages/outputarea/src/widget.ts

```diff
@@ -119,6 +119,17 @@ export class OutputArea extends Widget {
     ) {
       const output = model.get(i);
       this._insertOutput(i, output);
+      if (output.type === 'stream') {
+        // This is a stream output, follow changes to the text.
+        output.streamText!.changed.connect(
+          (
+            sender: IObservableString,
+            event: IObservableString.IChangedArgs
+          ) => {
+            this._setOutput(i, output);
```

Comment:
Is it safe to refer to numeric `i` here, or should it be some kind of stable reference?  
This runs later as output arrives/changes(?), meanwhile user could insert/delete cells above current cell?  [tried that, observed no bugs]

But hmm this is not indexing of cells, right?  Is it indexing lines of output?  Chunks of output of given stdout/stderr name?